src/xdocs/config.xml

```diff
@@ -293,6 +293,7 @@
       </subsection>
 
       <subsection name="Properties" id="Checker_Properties">
+        <div class="wrapper">
         <table>
```

Comment:
See for previous comment.

Reply:
answering to all conversations: some files had indents with a width of 2 spaces and some had for spaces. I can fix it if it is an issue.

Reply:
It is best to reply to each individual one to prevent confusion.

For here, why wasn't `table` indented at all?

Reply:
I seem to have missed this table. the problem with 4 and 2 spaces indents soesn't seem to be the reason for any of yout mentioned problems, sorry for wrong assumption.